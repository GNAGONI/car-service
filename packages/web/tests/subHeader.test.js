import React from 'react'
import renderer from 'react-test-renderer'
import SubHeader from '../components/subHeader'

it('renders correctly', () => {
  const tree = renderer.create(<SubHeader />).toJSON()
  expect(tree).toMatchSnapshot()
})
