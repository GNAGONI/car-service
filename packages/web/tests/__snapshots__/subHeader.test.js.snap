// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="main-header-content"
>
  <div
    className="icon-holder"
  >
    <img
      alt="description"
      height="97"
      src=""
      width="95"
    />
  </div>
  <div
    className="text-holder"
  >
    <h1>
      
    </h1>
    <p>
      
    </p>
  </div>
</div>
`;
